src/async_fuse/memfs/kv_engine.rs

```diff
@@ -180,23 +248,229 @@ pub trait MetaTxn {
     async fn commit(&mut self) -> DatenLordResult<bool>;
 }
 
+/// The option of 'set' operation
+/// Currently support 'lease' and `prev_kv`
+/// `lease` is used to set the lease of the key
+/// `prev_kv` is used to return the previous key-value pair
+#[allow(dead_code)]
```

Comment:
Please revisit the `[allow(dead_code)]` to make sure we really need it. We usually forget to remove it later.